searx/query.py

```diff
@@ -69,6 +70,19 @@ def parse_query(self):
                 self.query_parts.append(query_part)
                 continue
 
+            # this force the timeout
+            if query_part[0] == '<':
+                try:
+                    raw_timeout_limit = int(query_part[1:])
+                    if raw_timeout_limit < 10:
```

Comment:
Maybe adding a comment about seconds and centiseconds could help for the future.